src/Campaigns/Actions/LoadCampaignsListTableAssets.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Give\Campaigns\Actions;
+
+use Give\Campaigns\ListTable\CampaignsListTable;
+
+/**
+ * @unreleased
+ */
+class LoadCampaignsListTableAssets
```

Comment:
Why do we need these static functions that are only used here? Wouldn't it be better to have all this in the `__invoke` method, and add just hook up to an event? Of course, we will have to add a check for the `post_type` so that assets are loaded only on the campaigns page.

Reply:
Resolved in 274266fc4411b0e9904b517a22bcb7ec3e45526b